src/components/learning-guide.cjsx

```diff
@@ -130,9 +107,22 @@ LearningGuide = React.createClass
     guide = LearningGuideStore.get(@props.courseId)
     chapters = _.map(guide.children, @renderChapterPanels)
 
+    allSections = []
+    sections = guide.children
+    for section in sections
+      allSections.push(section.children)
+    allSections = _.flatten(allSections)
+
+    sortBest = _.sortBy(allSections, 'current_level')
+    sortBest = sortBest.reverse().slice(0, 5)
```

Comment:
@darrenmason I've just added a `LearningGuideStore.getSortedSections` method as part of #443 

With that you can: 
```coffee
all = LearningGuideStore.getSortedSections(@props.courseId)
page_ids = _.chain(sections).first(all, 4).pluck('page_ids').flatten().uniq().value()
@context.router.transitionTo('viewPractice', {courseId}, {page_ids})
```